generator/stdlib/src/jvmMain/kotlin/com/github/jcornaz/kwik/generator/stdlib/DateTime.kt

```diff
@@ -0,0 +1,63 @@
+package com.github.jcornaz.kwik.generator.stdlib
+
+import com.github.jcornaz.kwik.generator.api.Generator
+import java.time.Duration
+import java.time.Instant
+import java.time.LocalTime
+import java.util.*
+
+const val DAY_NANOSECONDS = 86400000000000L
```

Comment:
You can make this private ;-)
```suggestion
private const val DAY_NANOSECONDS = 86400000000000L
```